pennylane/ops/cv.py

```diff
@@ -750,15 +750,15 @@ class FockStateVector(CVOperation):
 
     **Details:**
 
-    * Number of wires: None (applied to the entire subsystem)
+    * Number of wires: Any
     * Number of parameters: 1
     * Gradient recipe: None (uses finite difference)
 
     Args:
         state (array): a single ket vector, for single mode state preparation,
             or a multimode ket, with one array dimension per mode
     """
-    num_wires = 0
+    num_wires = Any
     num_params = 1
     par_domain = "A"
     grad_method = "F"
```

Comment:
same comment about gradient method. Should it not be `None`?

Reply:
See https://github.com/XanaduAI/pennylane/pull/277#discussion_r307989136